lib/deimos/config/configuration.rb

```diff
@@ -451,7 +458,7 @@ def self.configure_producer_or_consumer(kafka_config)
       setting :bulk_import_id_column, :bulk_import_id
       # If true, multi-table consumers will blow away associations rather than appending to them.
       # @return [Boolean]
-      setting :replace_associations, true
+      setting :replace_associations, nil
```

Comment:
Can we add a test to validate this feature?

Reply:
Added a spec